rootfs/usr/bin/cec-control

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+ACTION="$1"
+
+
+# Source environment from ~/.config/environment.d with user overrides
+set -a
+for i in /home/gamer/.config/environment.d/*.conf ;
+do
+    [[ -f "${i}" ]] && . "${i}"
+done
+set +a
+
+# OVERRIDE WITH DEFAULTS IF NOT SET
```

Comment:
If a config file isn't present for this we should generate a default one and then read from it, so the user can identify how to change these behaviors. Hiding this in code without documentation or example would make using these as environment variables pointless.